src/sentry/api/endpoints/organization_releases.py

```diff
@@ -513,6 +513,23 @@ def post(self, request: Request, organization) -> Response:
                 raise ConflictError(
                     "Could not create the release it conflicts with existing data",
                 )
+
```

Comment:
Should this check happen before the try/except above?

Reply:
For new releases, projects are being added here below:
https://github.com/getsentry/sentry/blob/69398a5b11794764907ea54a262b139a34a7e64c/src/sentry/api/endpoints/organization_releases.py#L541-L542

and `projects` list contains only projects that a user has access to:
https://github.com/getsentry/sentry/blob/69398a5b11794764907ea54a262b139a34a7e64c/src/sentry/api/endpoints/organization_releases.py#L477-L489

If a user does not have access to a project, then the request fails with `Invalid project ids or slugs` before we create a release.